src/openforms/emails/digest.py

```diff
@@ -552,31 +551,30 @@ def collect_expired_or_near_expiry_referentielijsten_data():
                         service, client_factory=ReferentielijstenClient
                     ) as client:
                         try:
-                            response = client.get(
-                                "tabellen",
-                                params={"code": component["openForms"]["code"]},
-                            )
+                            table = client.get_tabel(component["openForms"]["code"])
                         except RequestException:
-                            response = []
+                            table = None
```

Comment:
this doesn't make sense to me - if you get a crash when retrieving the table, surely you'll want to report that in the digest email instead of continuing the rest of the checks?